src/components/Tooltip/TooltipTypes.d.ts

```diff
@@ -1,6 +1,18 @@
 import type { ElementType, ReactNode, CSSProperties, RefObject } from 'react'
 
-export type PlacesType = 'top' | 'right' | 'bottom' | 'left'
+export type PlacesType =
+  | "top"
+  | "right"
+  | "bottom"
+  | "left"
+  | "top-start"
+  | "top-end"
+  | "right-start"
+  | "right-end"
+  | "bottom-start"
+  | "bottom-end"
+  | "left-start"
+  | "left-end";
```

Comment:
just to be consistent with `floating-ui`

```suggestion
export type PlacesType =
  | 'top'
  | 'top-start'
  | 'top-end'
  | 'right'
  | 'right-start'
  | 'right-end'
  | 'bottom'
  | 'bottom-start'
  | 'bottom-end'
  | 'left'
  | 'left-start'
  | 'left-end'
```